app/assets/javascripts/admin/addon/components/admin-config-area-card.gjs

```diff
@@ -0,0 +1,16 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import i18n from "discourse-common/helpers/i18n";
+
+export default class AdminConfigAreaCard extends Component {
+  @tracked collapsed = false;
+
+  <template>
+    <section class="admin-config-area-card">
+      <h3>{{i18n @heading}}</h3>
```

Comment:
Suggest adding a CSS class to the title like `admin-config-area-card__title`

Reply:
done!